docs/docs/api/Dispatcher.md

```diff
@@ -817,6 +817,160 @@ try {
 }
 ```
 
+### `Dispatcher.compose(interceptors[, interceptor])`
```

Comment:
are those interceptors the same signature of the interceptor options?

Reply:
Yes, they should follow the same signature